inlong-sort/sort-flink/base/src/main/java/org/apache/inlong/sort/base/format/CanalJsonDynamicSchemaFormat.java

```diff
@@ -121,10 +123,14 @@ public JsonNode extractOperation(JsonNode data) {
     @Override
     public RowType extractSchema(JsonNode data, List<String> pkNames) {
         JsonNode schema = data.get(SCHEMA);
+        JsonNode dialectSchema = data.get(MYSQL_TYPE);
+        if (dialectSchema == null) {
+            dialectSchema = data.get(ORACLE_TYE);
```

Comment:
Are we only support mysql or oracle dialect?

Reply:
yes, only mysql and oracle dialect sql type are supported right now.

Other database dialect sql type could be here when cdc connector is ready.